docs/configuration/production.rst

```diff
@@ -205,7 +272,8 @@ Here's an example:
     [uwsgi]
     wsgi-file = app.wsgi
     enable-threads = true
-    http-socket = 127.0.0.1:8888
+    socket = /run/uwsgi/kinto.sock
```

Comment:
Question: Is this path cross-platform compatible? Also, I'd naturally expect it to be `/var/run`.